services/kvstoremanager/redis.go

```diff
@@ -26,7 +29,22 @@ func init() {
 	abortableErrors = []string{"connection refused", "invalid password"}
 }
 
-func (m *redisManagerT) Connect() {
+func NewRedisManager(config types.ConfigT) *RedisManager {
+	redisMgr := &RedisManager{
+		config: config,
+	}
+	redisMgr.Connect()
+	return redisMgr
+}
+
+func (m *RedisManager) GetClient() redis.Cmdable {
+	if m.clusterMode {
+		return m.clusterClient
+	}
+	return m.client
+}
+
+func (m *RedisManager) Connect() {
```

Comment:
This method doesn't connect to anything, in fact it doesn't return an error. If it would connect it would have to return one. I think we should rename it to `CreateClient()`.

Reply:
Makes sense

Reply:
Changes are done